### PR TITLE
Pic 473 attendance table content change

### PR DESCRIPTION
**Pull request by katie-cawthorne**
Following demo to Charan and Alastair needed to make a few more changes:
- Change heading to 'Attendance' not 'Appointment attendance"
- Improve alignment further to make it easier for the user to scan read the appointments

Made above changes and updated BDD tests.

### CODE REVIEW

File: server/views/case-summary-record-order.njk [hunk 1]
```diff
@@ -218,76 +221,85 @@
...
-                </p>
-
+                <p class="govuk-heading-m govuk-!-margin-bottom-1"><span
+                            class="pac-dashboard-count">{{ absentAppointments.failed.flat(Infinity) | length }}</span><span
+                            class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block">
+                Absences </span></p>
                 {% for failed in absentFailed | sort(true, false, 'count') %}
                     <p class="govuk-body govuk-!-margin-0"><span
-                                class="govuk-body govuk-!-margin-right-2 app-dashboard-count">{{ failed.count }}</span> {{ failed.description }}
+                                class="govuk-body- pac-dashboard-count">{{ failed.count }}</span>{{ failed.description }}
```
Comment (theDustRoom) [inline server/views/case-summary-record-order.njk:302]: There's an additional - after govuk-body

File: server/views/case-summary-record-order.njk [hunk 2]
```diff
@@ -218,76 +221,85 @@
...
-                </p>
-
+                <p class="govuk-heading-m govuk-!-margin-bottom-1"><span
+                            class="pac-dashboard-count">{{ absentAppointments.complied.flat(Infinity) | length }}</span><span
+                            class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-0 govuk-!-display-inline-block">
+                Absences </span></p>
                 {% for complied in absentComplied | sort(true, false, 'count') %}
                     <p class="govuk-body govuk-!-margin-0"><span
-                                class="govuk-body govuk-!-margin-right-2 app-dashboard-count">{{ complied.count }}</span> {{ complied.description }}
+                                class="govuk-body- pac-dashboard-count">{{ complied.count }}</span>{{ complied.description }}
```
Comment (theDustRoom) [inline server/views/case-summary-record-order.njk:272]: There's an additional - after govuk-body

**Review by theDustRoom — CHANGES_REQUESTED** [review]
Just a couple of class names including erroneous _**-**_ that need fixing

File: server/views/case-summary-record-order.njk (hunk 2 above)
Comment (katie-cawthorne) [reply server/views/case-summary-record-order.njk:272]: Yep I'll sort it, good spot

File: server/views/case-summary-record-order.njk (hunk 1 above)
Comment (katie-cawthorne) [reply server/views/case-summary-record-order.njk:302]: Good spot, I'll sort it

**Review by theDustRoom — APPROVED** [review]
LGTM 👍

**Pull request merged**